_faq/writing-a-great-proposal.md

```diff
@@ -21,7 +21,7 @@ lot of helpful feedback from us!
 
 #### Writing the proposal
 
-Please follow the **application template** in the faq. All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
+Please follow the **application template[application template](https://raw.githubusercontent.com/coala/projects/master/_faq/application-template.md)** in the faq. All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
```

Comment:
When the user clicks on the anchor, he should be directed to the above appication template tab on the same webpage, not the .md file. Please click on netlify preview to have a look at the changes whenever you push :wink: